mkosi/initrd.py

```diff
@@ -142,6 +179,7 @@ def initrd_finalize(staging_dir: str, output: str, output_dir: str) -> None:
 def initrd_common_args(parser: argparse.ArgumentParser) -> None:
     parser.add_argument(
         "--kernel-version",
+        "-k",
```

Comment:
Could you move that before the long option? The short ones are first in the other parser definitions.